README.md

```diff
@@ -81,17 +100,17 @@ The observation of the agent is also configuable, including internal states, tar
 
     <img src="media/reaching.gif" width="320" height="240" alt="reaching"/>
 
-* [PickAndPlace task](python/social_bot/tasks.py): A task to grip an object (a wood cube), move and then place it to the target position. An optional reward shaping can be used to guide the agent to grasp cube and move to the position. This task is only compatible with Agent youbot_noplugin.
+* [PickAndPlace task](python/social_bot/tasks.py): A task to grip an object (a wood cube), move and then place it to the target position. An optional reward shaping can be used to guide the agent to grasp cube and move to the position. This task is only compatible with Agent youbot_noplugin. Check for [pickplace_sac_youbot.gin](examples/pickplace_sac_youbot.gin) for a training example using ALF.
```

Comment:
If you still have those training curves, can you add them to the example directory so that people can know whether the training is correct if they want to run the examples in the future.